lib/FieldType/Mapper/CountryFormMapper.php

```diff
@@ -8,24 +8,22 @@
  */
 namespace EzSystems\RepositoryForms\FieldType\Mapper;
 
+use EzSystems\RepositoryForms\Data\Content\FieldData;
 use EzSystems\RepositoryForms\Data\FieldDefinitionData;
-use EzSystems\RepositoryForms\FieldType\DataTransformer\CountryValueTransformer;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\MultipleCountryValueTransformer;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\SingleCountryValueTransformer;
 use EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface;
+use EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface;
 use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CountryFormMapper implements FieldDefinitionFormMapperInterface
+class CountryFormMapper implements FieldDefinitionFormMapperInterface, FieldValueFormMapperInterface
 {
-    /**
-     * @var array Array of countries from ezpublish.fieldType.ezcountry.data
-     */
+    /** @var array Array of countries from ezpublish.fieldType.ezcountry.data */
     protected $countriesInfo;
 
-    /**
-     * @param array $countriesInfo
-     */
```

Comment:
Why was this doc removed?

Reply:
Might be because the type hint and param name is already in the php code, so as long as phpdoc can't add more info than wheat the code does some people tend to prefer removing it.

Reply:
Ack, would prefer adding a description then instead, unless that's not really needed.

Reply:
true, but it's a ctor, it's usually self describing

Reply:
Actually I haven't intended to remove that phpdoc, removed it by mistake. Fixed it :)

Reply:
Actually I agreed with @andrerom. Without added information, it's mostly an extra risk that we end-up with mismatching info whenever something changes. But let's not touch that now.